ubuntu_install_requirements.sh

```diff
@@ -19,6 +19,7 @@ sudo apt -y update
 
 sudo apt -y install \
   python-pip \
+  python3-pip \
```

Comment:
Do you need to install this here, or can it just be installed via ansible (since the package install happens before any pip installs):

https://github.com/metal3-io/metal3-dev-env/blob/master/vm-setup/roles/packages_installation/tasks/main.yml#L5

Reply:
Ah I see, so in a future commit https://github.com/metal3-io/metal3-dev-env/blob/master/ubuntu_install_requirements.sh#L91 will get switched to pip3 - thanks for clarifying

Reply:
Yes apparently